src/app/components/JumpTo/index.tsx

```diff
@@ -84,6 +83,26 @@ const JumpTo = ({ jumpToHeadings }: JumpToProps) => {
             </li>
           );
         })}
+        {showRelatedContentLink && (
+          <li key="#related-content-heading" css={styles.listItem}>
+            <a
+              href="#related-content-heading"
+              onClick={e => linkClickHandler(e, '#related-content-heading')}
+              css={styles.link}
+              data-testid="jump-to-link-related-content-heading"
+            >
+              <span
+                css={[
+                  styles.linkText,
+                  decodeURIComponent(hash) === '#related-content-heading' &&
+                    styles.linkTextActive,
+                ]}
+              >
+                {relatedContent}
+              </span>
+            </a>
+          </li>
```

Comment:
Would it make sense / be easier to update the jumpToHeadings list to include the related content as a new entry, as this seems like duplication to me.

Could do something along the lines of (pseudocode):

```
const jumpToHeadingsWithId = jumpToHeadings.map(({heading}) => {
 const sanitisedId = idSanitiser(heading);
 const idWithHash = `#${sanitisedId}`;

 return { 
    heading, 
    idWithHash
 }
});

if (showRelatedContent) {
   jumpToHeadingsWithId.push({ heading: relatedContent, idWithHash: '#related-content-heading'})
}

// Now map over the jumpToHeadingsWithId and render
// code from lines 66-83
```

Reply:
I did this in an earlier version of this code. You and @amoore108 have different ideas (both work!), which shall I go with? 🧠

Reply:
Although it is a bit different actually... maybe it can be a combination of both of your ideas in a way

Reply:
Seems like a good combination. Should be a 3 liner to add most read when it comes to it with that approach.

It doesn't mutate the data either as `.map` will create a new array, so its all good from that perspective.

Reply:
it works. I need to change some tests to match this.